### PR TITLE
Updated ItemsRepeater Page

### PR DESCRIPTION
**Pull request by anawishnoff**
## Description
Added two new samples: one sample that shows an ItemsRepeater with mixed-type items, and another that shows how to use nested ItemsRepeaters.

I also updated the sample code for all examples making them more detailed, and adding C# code-behind samples when relevant.

## Motivation and Context

This is a part of my ongoing changes to the collections/layout pages of the Controls Gallery to make the samples easier to use for new developers, yet still helpful and interesting to experienced developers. For ItemsRepeater specifically, I wanted to keep the samples centered around interesting layout situations while showing off the flexibility of the control.

## How Has This Been Tested?

Manually tested by me in dark mode and light mode, tried all possible combinations/options, etc.

## Screenshots (if appropriate):
itemsrepeaterCG
itemsrepeaterCG2

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml [hunk 1]
```diff
@@ -42,6 +42,49 @@
             </Button>
         </DataTemplate>
 
+        <DataTemplate x:Key="StringDataTemplate" x:DataType="x:String">
+            <Grid Background="{StaticResource SystemControlBackgroundAccentBrush}" Margin="10">
+                <TextBlock Padding="10" Text="{x:Bind}" Foreground="{StaticResource SystemControlForegroundAltHighBrush}" HorizontalAlignment="Center" TextWrapping="Wrap" VerticalAlignment="Center"></TextBlock>
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml]: On an accent-colored background, we typically recommend showing white text regardless of light/dark theme. So SystemControlForegroundChromeWhiteBrush seems like a better choice.

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml [hunk 2]
```diff
@@ -42,6 +42,49 @@
...
+
+        <DataTemplate x:Key="StringDataTemplateColor" x:DataType="x:String">
+            <Grid Background="{StaticResource SystemControlBackgroundAccentBrush}" Margin="10, 0, 10, 10">
+                <TextBlock Padding="10" Text="{x:Bind}" Foreground="{StaticResource SystemControlForegroundAltHighBrush}" HorizontalAlignment="Center" TextWrapping="Wrap" VerticalAlignment="Center"></TextBlock>
+            </Grid>
+        </DataTemplate>
+
+        <DataTemplate x:Key="IntDataTemplate" x:DataType="x:Int32">
+            <Grid Background="{ThemeResource SystemChromeLowColor}" Margin="10">
+                <TextBlock Padding="10" Text="{x:Bind}" Foreground="{StaticResource SystemControlForegroundBaseHighBrush}" FontSize="50" HorizontalAlignment="Center" VerticalAlignment="Center"></TextBlock>
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml]: This is the default color, so you don't need to specify the Foreground. Also, let's stay within existing font sizes and use HeaderTextBlockStyle. 
And finally, would you mind fixing your TextBlock declarations to end with a closing bracket instead of a hanging tag? So <TextBlock ... /> instead of <TextBlock …></TextBlock>

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml [hunk 3]
```diff
@@ -42,6 +42,49 @@
...
+            </Grid>
+        </DataTemplate>
+
+        <DataTemplate x:Key="ImageTemplate" x:DataType="x:String">
+            <StackPanel>
+                <Image Height="200" Source="{x:Bind}" Margin="5"></Image>
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml]: Similar comment for Image, and really all other controls :)
```suggestion
                <Image Height="200" Source="{x:Bind}" Margin="5"/>
```

File: XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample1_xaml.txt [hunk 4]
```diff
@@ -0,0 +1,44 @@
+<!-- XAML Code -->
+
+<!-- This is the ItemsRepeater used in this example: -->
+<ItemsRepeater x:Name="MixedTypeRepeater" 
+               Margin="0,0,12,0"
+               HorizontalAlignment="Stretch"
+               ItemTemplate="{StaticResource StringOrIntTemplateSelector}">
+
+    <ItemsRepeater.Layout>
+        <UniformGridLayout MinItemWidth="200" MinItemHeight="200"></muxcontrols:UniformGridLayout>
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample1_xaml.txt]: Just realized that, for consistency, all mentions in sample code of ItemsRepeater or UniformGridLayout should have the muxc namespace prefix. Probably also StackLayout - anything that's defined in WinUI.

```suggestion
        <muxc:UniformGridLayout MinItemWidth="200" MinItemHeight="200"/>
```

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml [hunk 5]
```diff
@@ -32,16 +32,53 @@
...
+        </DataTemplate>
+
+        <DataTemplate x:Key="StringDataTemplateColor" x:DataType="x:String">
+            <Grid Background="{ThemeResource SystemControlBackgroundAccentBrush}" Margin="10, 0, 10, 10">
+                <TextBlock Padding="8" Text="{x:Bind}" Foreground="{ThemeResource SystemControlForegroundChromeWhiteBrush}" HorizontalAlignment="Center" VerticalAlignment="Center"/>
+            </Grid>
+        </DataTemplate>
+
+        <DataTemplate x:Key="IntDataTemplate" x:DataType="x:Int32">
+            <Grid Background="{ThemeResource SystemChromeLowColor}" Margin="10">
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml]: The background switched from a brush to a color resource. Brushes are better because they correctly handle high contrast.

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml [hunk 6]
```diff
@@ -129,18 +194,95 @@
...
+        ItemTemplate ="{StaticResource CategoryTemplate}"&gt;
+        &lt;ItemsRepeater.Layout&gt;
+           &lt;StackLayout Orientation="Vertical" /&gt;
+        &lt;/ItemsRepeater.Layout&gt;
+    &lt;/ItemsRepeater&gt;
+&lt;/ScrollViewer&gt;
+                    
+&lt;DataTemplate x:Key="CategoryTemplate" x:DataType="l:NestedCategory"&gt;
+    &lt;StackPanel&gt;
+        &lt;TextBlock Text="{x:Bind CategoryName}" Padding="8" FontSize="30"/&gt;
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml]: Don't forget to update the sample to match the actual code :)

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs [hunk 7]
```diff
@@ -42,6 +47,78 @@ private void InitializeData()
...
+                                                            "Eggs"
+                }));
+
+            outerRepeater.ItemsSource = nestedCategories;
+
+            // Intitalize code sample for non-interactive items layout
+            displayLayout.Value = "<StackLayout x:Name='VerticalStackLayout' Orientation='Vertical' Spacing='8'/>";
+
+            displayDataTemplate.Value = @"<DataTemplate x:Key='HorizontalBarTemplate' x:DataType='l: Bar'>
+    <Border Background = '{ThemeResource SystemChromeLowColor}' Width = '{x:Bind MaxLength}' >
```
[the remark this answers is not in the document]
Comment (anawishnoff) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: Getting syntax errors when trying to escape " like this?

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs [hunk 8]
```diff
@@ -42,6 +47,78 @@ private void InitializeData()
...
+        <Rectangle Fill = '{ThemeResource SystemAccentColor}' Width = '{x:Bind Length}' 
+                   Height = '24' HorizontalAlignment = 'Left'/>
```
[the remark this answers is not in the document]
Comment (anawishnoff) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: Which spaces? 😅

File: XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample1_xaml.txt (hunk 4 above)
Comment (anawishnoff) [reply XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample1_xaml.txt]: Fixed all.

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs (hunk 7 above)
[the remark this answers is not in the document]
Comment (YuliKl) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: Maybe try using an @ symbol instead of \ ?

[the remark this answers is not in the document]
Comment (YuliKl) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: https://stackoverflow.com/questions/14480724/escape-double-quotes-in-string

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs (hunk 8 above)
[the remark this answers is not in the document]
Comment (YuliKl) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: Mostly around the =

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs [hunk 9]
```diff
@@ -1,11 +1,15 @@
...
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Diagnostics;
 using System.IO;
 using System.Linq;
 using System.Runtime.InteropServices.WindowsRuntime;
 using Windows.Foundation;
 using Windows.Foundation.Collections;
+using Windows.Services.Maps.OfflineMaps;
```
Comment (stmoy) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: ? Please remove unnecessary `using`s.

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs [hunk 10]
```diff
@@ -42,6 +47,78 @@ private void InitializeData()
...
+            basicData.Add(128);
+            basicData.Add("Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.");
+            basicData.Add(256);
+            basicData.Add("Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.");
+            basicData.Add(512);
+            basicData.Add("Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.");
+            basicData.Add(1024);
+            MixedTypeRepeater.ItemsSource = basicData;
+
+            ObservableCollection<NestedCategory> nestedCategories = new ObservableCollection<NestedCategory>();
```
Comment (stmoy) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: NIT: Do these need to be `ObservableCollection`? Probably not, since the collections aren't changing during runtime.

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs [hunk 11]
```diff
@@ -42,6 +47,78 @@ private void InitializeData()
...
+                                                            "Salmon",
+                                                            "Pork",
+                                                            "Chickpeas",
+                                                            "Eggs"
+                }));
+
+            outerRepeater.ItemsSource = nestedCategories;
+
+            // Intitalize code sample for non-interactive items layout
+            displayLayout.Value = "<muxc:StackLayout x:Name='VerticalStackLayout' Orientation='Vertical' Spacing='8'/>";
```
Comment (stmoy) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: This is a strange mixing of xaml in the codebehind. Why is this here? Generally folks will keep their XAML in the .xaml file to help enable MVVM scenarios and ensure that concerns are separated. Same with the below.

File: XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample1_cs.txt [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+    public DataTemplate IntTemplate { get; set; }
+
+    protected override DataTemplate SelectTemplateCore(object item)
+    {
+        // Return the correct data template based on the item's type.
+        if (item.GetType() == typeof(String))
+        {
+            return StringTemplate;
+        }
+
```
Comment (stmoy) [inline XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample1_cs.txt]: Please ensure that these .txt files are updated when you respond to the PR feedback above.

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml [hunk 13]
```diff
@@ -129,18 +188,89 @@
...
                 <local:ControlExampleSubstitution Key="ItemTemplate" Value="SimpleItemTemplate" />
+                <local:ControlExampleSubstitution x:Name="displayLayout2" Key="displayLayout2"/>
             </local:ControlExample.Substitutions>
         </local:ControlExample>
+        
+        <local:ControlExample HeaderText="ItemsRepeater with mixed-type collection"
+                              XamlSource="ItemsRepeater/ItemsRepeaterSample1_xaml.txt"
+                              CSharpSource="ItemsRepeater/ItemsRepeaterSample1_cs.txt">
+            <StackPanel>
+                <TextBlock Text="This is an ItemsRepeater that displays both integer and string items. It uses a DataTemplateSelector to choose the correct layout for each of its items. The full source code for this ItemsRepeater is below." TextWrapping="Wrap"/>
```
Comment (stmoy) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml]:
```suggestion
                <TextBlock Text="This is an ItemsRepeater that displays both integer and string items. It uses a DataTemplateSelector to choose the correct layout for each of its items." TextWrapping="Wrap"/>
```

It is assumed that the code is shown below.

**Comment by stmoy** [comment]
We also briefly chatted about moving this page to the Collections section of the Gallery. It looks like it is still under Layout. Double checking if you decided one way or the other.

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs (hunk 10 above)
Comment (anawishnoff) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: Changed to `List`.

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs (hunk 11 above)
Comment (anawishnoff) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: So this is the way that @YuliKl used in the NavigationView pages to dynamically update the sample code section based on buttons that were clicked on the page. Since this needs more logic than the '$()' notation allows within the ControlExample.Xaml, this seems like the best way to do it.

**Comment by anawishnoff** [comment]
> We also briefly chatted about moving this page to the Collections section of the Gallery. It looks like it is still under Layout. Double checking if you decided one way or the other.

After discussing the samples with @pag3, we decided that keeping the ItemsRepeater page in the Layout section would be best as its more of a building block then a standalone collection control. 

> Additionally: I really like the look of the new samples. You could consider moving them to the top of the page.
He also suggested this specific ordering because it told the best story, in terms of how the features tie into one another/the ascending complexity of each sample.

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs (hunk 11 above)
Comment (stmoy) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: Oh! I think I misunderstood the comment and intent. To clarify: these chunks of XAML are just to support swapping out the sample code? If so, that's fine.

Comment (anawishnoff) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: Yep 👍 Should i change the comment to clarify?

Comment (stmoy) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: Please do. Either something like "Set the sample code displayed on the page" or something. When I saw "initialize" and the name of the variable is "displayLayout", I thought we were doing some swapping out of XAML elements.

You could also consider renaming "displayLayout" to something like "sampleCodeLayout" or something that makes it clear that this isn't actually functional code.

Comment (anawishnoff) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: Done. All your changes should be in place!

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml [hunk 14]
```diff
@@ -32,16 +32,47 @@
...
+            <Grid Background="{ThemeResource SystemControlBackgroundAccentBrush}" Margin="10">
+                <TextBlock Padding="10" Text="{x:Bind}" Foreground="{ThemeResource SystemControlForegroundChromeWhiteBrush}" HorizontalAlignment="Center" TextWrapping="Wrap" VerticalAlignment="Center"/>
+            </Grid>
+        </DataTemplate>
+
+        <DataTemplate x:Key="StringDataTemplateColor" x:DataType="x:String">
+            <Grid Background="{ThemeResource SystemControlBackgroundAccentBrush}" Margin="10, 0, 10, 10">
+                <TextBlock Padding="8" Text="{x:Bind}" Foreground="{ThemeResource SystemControlForegroundChromeWhiteBrush}" HorizontalAlignment="Center" VerticalAlignment="Center"/>
+            </Grid>
+        </DataTemplate>
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml]: These are **so** similar, feels kind of redundant to have both. I guess the Grid's top margin the only real differentiation.

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs [hunk 15]
```diff
@@ -42,6 +38,78 @@ private void InitializeData()
...
+                                                            "Chickpeas",
+                                                            "Eggs"
+                }));
+
+            outerRepeater.ItemsSource = nestedCategories;
+
+            // Set sample code to display on page's initial load
+            SampleCodeLayout.Value = "<muxc:StackLayout x:Name='VerticalStackLayout' Orientation='Vertical' Spacing='8'/>";
+
+            SampleCodeDT.Value = @"<DataTemplate x:Key=""HorizontalBarTemplate"" x:DataType=""l: Bar"">
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]:
```suggestion
            SampleCodeDT.Value = @"<DataTemplate x:Key=""HorizontalBarTemplate"" x:DataType=""l:Bar"">
```

I believe the extra space would prevent that code from working.

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs [hunk 16]
```diff
@@ -42,6 +38,78 @@ private void InitializeData()
...
+                                                            "Salmon",
+                                                            "Pork",
+                                                            "Chickpeas",
+                                                            "Eggs"
+                }));
+
+            outerRepeater.ItemsSource = nestedCategories;
+
+            // Set sample code to display on page's initial load
+            SampleCodeLayout.Value = "<muxc:StackLayout x:Name='VerticalStackLayout' Orientation='Vertical' Spacing='8'/>";
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: These need to be double quotes instead of single. The purpose of the samples is allow people using the Gallery to copy-paste them into their own apps and use as is. So the sample code needs to show correct syntax.
```suggestion
            SampleCodeLayout.Value = "<muxc:StackLayout x:Name=\"VerticalStackLayout\" Orientation=\"Vertical\" Spacing=\"8\"/>";
```

Comment (YuliKl) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: Interestingly, this only looks wrong when the page loads. If I click on one of the other layout radio buttons and go back to this layout, everything looks good. Are you loading the sample code from different places?

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs [hunk 17]
```diff
@@ -107,22 +186,47 @@ private void RadioBtn_Click(object sender, RoutedEventArgs e)
                 itemTemplateKey = "HorizontalBarTemplate";
 
                 repeater.MaxWidth = MaxLength + 12;
+
+                SampleCodeLayout.Value = @"<muxc:StackLayout x:Name=""VerticalStackLayout"" Orientation=""Vertical"" Spacing=""8""/>";
+                SampleCodeDT.Value = @"<DataTemplate x:Key=""HorizontalBarTemplate"" x:DataType=""l: Bar"">
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]:
```suggestion
                SampleCodeDT.Value = @"<DataTemplate x:Key=""HorizontalBarTemplate"" x:DataType=""l:Bar"">
```

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs [hunk 18]
```diff
@@ -107,22 +186,47 @@ private void RadioBtn_Click(object sender, RoutedEventArgs e)
...
             else if (layoutKey.Equals(nameof(this.UniformGridLayout)))
             {
                 layout.Value = layoutKey;
                 itemTemplateKey = "CircularTemplate";
 
                 repeater.MaxWidth = 540;
-            }
 
+                SampleCodeLayout.Value = @"<muxc:UniformGridLayout x:Name=""UniformGridLayout"" MinRowSpacing=""8"" MinColumnSpacing=""8""/>";
+                SampleCodeDT.Value = @"<DataTemplate x:Key=""CircularTemplate"" x:DataType=""l: Bar"">
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]:
```suggestion
                SampleCodeDT.Value = @"<DataTemplate x:Key=""CircularTemplate"" x:DataType=""l:Bar"">
```

File: XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample1_xaml.txt [hunk 19]
```diff
@@ -0,0 +1,44 @@
+<!-- XAML Code -->
+
+<!-- This is the ItemsRepeater used in this example: -->
+<muxc:ItemsRepeater x:Name="MixedTypeRepeater" 
+               Margin="0,0,12,0"
+               HorizontalAlignment="Stretch"
+               ItemTemplate="{StaticResource StringOrIntTemplateSelector}">
+
+    <ItemsRepeater.Layout>
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample1_xaml.txt]:
```suggestion
    <muxc:ItemsRepeater.Layout>
```

File: XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample1_xaml.txt [hunk 20]
```diff
@@ -0,0 +1,44 @@
...
+        <muxc:UniformGridLayout MinItemWidth="200" MinItemHeight="200"></muxc:UniformGridLayout>
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample1_xaml.txt]:
```suggestion
        <muxc:UniformGridLayout MinItemWidth="200" MinItemHeight="200"/>
```

File: XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample1_xaml.txt [hunk 21]
```diff
@@ -0,0 +1,44 @@
...
+    </ItemsRepeater.Layout>
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample1_xaml.txt]:
```suggestion
    </muxc:ItemsRepeater.Layout>
```

File: XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterNestedSample_xaml.txt [hunk 22]
```diff
@@ -0,0 +1,41 @@
+<!-- The nested ItemsRepeater experience is achieved by creating one ItemsRepeater (outerRepeater below) 
+and creating another ItemsRepeater (innerRepeater below) inside the outer one's DataTemplate. Below is 
+the outer ItemsRepeater definition and the DataTemplate it uses. -->
+                    
+<ScrollViewer>
+    <muxc:ItemsRepeater 
+        x:Name="outerRepeater"
+        VerticalAlignment="Top"
+        ItemTemplate ="{StaticResource CategoryTemplate}">
+        <ItemsRepeater.Layout>
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterNestedSample_xaml.txt]:
```suggestion
        <muxc:ItemsRepeater.Layout>
```

File: XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterNestedSample_xaml.txt [hunk 23]
```diff
@@ -0,0 +1,41 @@
...
+           <muxc:StackLayout Orientation="Vertical" />
+        </ItemsRepeater.Layout>
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterNestedSample_xaml.txt]:
```suggestion
        </muxc:ItemsRepeater.Layout>
```

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs (hunk 16 above)
Comment (anawishnoff) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs]: Fixed. This code block populates the samples when the page loads, I must have missed it when replacing all the single quotes to double quotes.
